test/entropic/sinkhorn_divergence.jl

```diff
@@ -53,17 +53,17 @@ Random.seed!(100)
             ν = hcat([normalize!(f.(x; μ=randn(), σ=0.5), 1) for _ in 1:M]...)
             for reg in (true, false)
                 loss_batch = sinkhorn_divergence(μ, ν, C, ε; regularization=reg)
-                @test loss_batch ≈ [
+                @compat @test loss_batch ≈ [
```

Comment:
Same as above, I think it would be preferable to rewrite the code. In any case `@compat` is not needed.